doc/tutorials/make_your_first_quantum_circuit.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>Might be worth including an 'important' box, explaining why this is the case?</p><p><br></p><p>Could be done like this, in a new Markdown cell:</p><p><br></p><p>&lt;div class="admonition important"&gt;</p><p>&lt;div class="admonition-title"&gt;Important&lt;/div&gt;</p><p>When constructing a hybrid quantum/classical computational model with PennyLane, it is important to always import NumPy from PennyLane, not the standard NumPy!</p><p>&lt;/div&gt;</p><p>&lt;/div&gt;</p><p><br></p><p>This will render much more nicely in the documentation! In terms of the allowed admonitions, you can do:</p><p><br></p><ul><li>note, hint: blue background</li><li>aside: green background</li><li>important, warning, attention, caution: yellow background</li><li>danger, error: red background</li></ul>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good idea! Thanks for this! I will update other notebooks too where necessary :)

Reply:
It does get highlighted inside the jupyter notebook itself though.